Platform/TigerlakeBoardPkg/BoardConfig.py

```diff
@@ -289,6 +290,11 @@ def GetPlatformDsc (self):
             'TopSwapLib|Silicon/CommonSocPkg/Library/TopSwapLib/TopSwapLib.inf'
         ]
 
+        if self.TXT_ENABLED:
+            dsc['LibraryClasses.%s' % self.BUILD_ARCH].append ('TxtLib|Silicon/$(SILICON_PKG_NAME)/Library/TxtLib/TxtLib.inf')
+        else:
+            dsc['LibraryClasses.%s' % self.BUILD_ARCH].append ('TxtLib|Silicon/$(SILICON_PKG_NAME)/Library/TxtLib/TxtLibNull.inf')
```

Comment:
It would be better to move TxtLibNull to Silicon/CommonSocPkg/Library. This way, if other platforms require the TXT feature in the future, they can leverage the same TxtLibNull.